src/Security/Security.Test/ScenarioTests/SqlVulnerabilityAssessmentTests.ps1

```diff
@@ -19,48 +19,48 @@
 	New-AzMonitorLogAnalyticsSolution cmdlet depends on Az.MonitoringSolution
 	which is a generated module and is not currently supported by the testing framework
 #>
-function Test-AzSecuritySqlVulnerabilityAssessment
+function Test-AzSecuritySecuritySqlVulnerabilityAssessment
```

Comment:
```suggestion
function Test-AzSecuritySqlVulnerabilityAssessment
```

Reply:
these were fixed yesterday with the "fix rename issues from tests.", please look at the latest version